conductor-client/src/main/java/com/openlattice/organizations/roles/SecurePrincipalsManager.java

```diff
@@ -128,18 +116,6 @@ void createSecurablePrincipal(
 
     Map<SecurablePrincipal, Set<Principal>> bulkGetUnderlyingPrincipals( Set<SecurablePrincipal> sps );
 
-    Collection<Principal> getAllUnderlyingPrincipals( SecurablePrincipal sp );
-
-    /**
-     * Returns all Principals, which have all the specified permissions on the securable object
-     *
-     * @param key         The securable object
-     * @param permissions Set of permission to check for
-     */
-    Set<Principal> getAuthorizedPrincipalsOnSecurableObject( AclKey key, EnumSet<Permission> permissions );
-
-    SecurablePrincipal getSecurablePrincipalById( UUID id );
-
```

Comment:
I get that these are not currently used but a couple of them seem helpful to have around, `getAuthorizedPrincipalsOnSecurableObject` and `getSecurablePrincipalById`. 
Are there other calls that effectively replace the need for these two?

Reply:
Yep -- getAuthorizedPrincipalsOnSecurableObject is in AuthorizationsManager (better there than PrincipalsManager), and getSecurablePrincipalById moved to PrinicpalMapManager (and is by AclKey, not id)